_generated/AzureTestPlanV0_Node20/task.json

```diff
@@ -36,11 +36,11 @@
     },
     {
       "name": "testPlan",
-      "type": "pickList",
+      "type": "string",
       "label": "Test plan",
       "defaultValue": "",
       "required": true,
-      "helpMarkDown": "Select a test plan containing test suites with test cases.",
+      "helpMarkDown": "Type or paste the test plan ID containing test suites with test cases.",
```

Comment:
Instead of Type or Paste, should we say Enter. If this is what PM has suggested, ignore my comment

Reply:
Yes, Igor shared this message for helpMarkdown